tools/claude-review/scripts/mdsafe.py

```
@@ -0,0 +1,132 @@
+# 最初の 1 文字だけを見る。
+_LEADING_STRUCT = re.compile(r"^( {0,3})([#>\-+*`~=_])")
+
+# 番号付きリストは記号 1 文字ではなく「数字列 + `.`/`)`」がマーカーになる。
+# CommonMark はバックスラッシュで数字自体をエスケープしても効かない
+# （`\1` はそのまま `\1` と解釈される）ため、区切り文字（`.`/`)`）の
+# 直前にバックスラッシュを置いて区切り文字自体を無害化する。
+_LEADING_ORDERED = re.compile(r"^( {0,3})(\d+)([.)])")
+
+# @mention 無害化用のゼロ幅スペース(U+200B)。「@」の直後に挿入し、
+# 見た目を変えずに「@username」としての文字の連続性だけを断つ。
+_ZWSP = "​"
```

Ruff (0.16.3)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)